### PR TITLE
added hw-10-1

### PR DESCRIPTION
**Pull request by kenu21**

### CODE REVIEW

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 1]
```diff
@@ -17,7 +26,23 @@
...
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        InputStream inputStream = Files.newInputStream(Paths.get(fileName));
+        int bytesOfFile;
+        StringBuilder onlyLetters = new StringBuilder();
+        do {
+            bytesOfFile = inputStream.read();
+            if ((bytesOfFile > 64 && bytesOfFile < 91) || (bytesOfFile > 96 && bytesOfFile < 123)) {
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: @kenu21 what means 64? 91? 96? 123? Why this numbers?

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 2]
```diff
@@ -17,7 +26,23 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        InputStream inputStream = Files.newInputStream(Paths.get(fileName));
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: @kenu21 please use try with resources

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 1 above)
Comment (boroda4436) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: @kenu21 you may do this more simpler

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 1 above)
Comment (kenu21) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: I created final int. Sofia said me it today - isLetter(). Can I do it in my way?

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 2 above)
Comment (kenu21) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: OK.

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 3]
```diff
@@ -17,7 +26,31 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        final int numberLetters = 5;
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: better move this variables in the class and make them as constants

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 4]
```diff
@@ -17,7 +26,31 @@
...
+        final int letter_A = 65;
+        final int Letter_Z = 90;
+        final int letter_a = 97;
+        final int letter_z = 122;
+
+        try (InputStream inputStream = Files.newInputStream(Paths.get(fileName))) {
+            int bytesOfFile;
+            StringBuilder onlyLetters = new StringBuilder();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: can we start to use treeset here?

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 3 above)
Comment (kenu21) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: OK. I'll fix it in a new commit.

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 4 above)
Comment (kenu21) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: I think, no. I have to use lowerCase().

Comment (kenu21) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: @Sofasmile No.

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 5]
```diff
@@ -17,7 +25,31 @@
...
+            Iterator alphabet = chars.iterator();
+            StringBuilder result = new StringBuilder("");
+            for (int i = 0; i < NUMBER_LETTERS && alphabet.hasNext(); i++) {
+                result.append(alphabet.next());
+            }
+            return result.toString();
+        }
+    }
+
+    private boolean isLatChar(char character) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]:
```suggestion
    private boolean isLatterChar(char character) {
```

**Review by Sofasmile — APPROVED** [review]
good job) make small fix

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 5 above)
Comment (kenu21) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: Thanks a lot!) Done.

**Pull request closed without merging**